servers/src/main/java/tachyon/worker/lineage/LineageDataManager.java

```diff
@@ -123,6 +114,29 @@ public synchronized void persistFile(long fileId, List<Long> blockIds) throws IO
     mPersistedFiles.add(fileId);
   }
 
+  /**
+   * Gets the destination file path of the given file id. Also creates the parent folder if it does
```

Comment:
it is somehow weird for a seemingly getter method to have side-effect (creating parent folder). Either rename this function more meaningful or separate logic?